docs/google/fetch-logs.md

```diff
@@ -0,0 +1,66 @@
+|[Axeman](https://github.com/CaliDog/Axeman)                                                            |files (csv)                  |[yes](https://github.com/CaliDog/Axeman/blob/e8a195a3e31f10ee6156d564ec541e7dcc356a4c/axeman/core.py#L28)                                |[no](https://github.com/CaliDog/Axeman/blob/e8a195a3e31f10ee6156d564ec541e7dcc356a4c/axeman/certlib.py#L60)                       |no                                                                                                                          |
+|[ctmon](https://github.com/sergiogarciadev/ctmon)                                                      |PostgreSQL                   |[yes](https://github.com/search?q=repo%3Asergiogarciadev%2Fctmon+path%3Astate.json+concurrency&type=code)                                |[yes](https://github.com/sergiogarciadev/ctmon/blob/e4a4f67f4b405821a2ab47ab1878d6ae0eebb72c/logclient/log.go#L73)                |[randomized](https://github.com/sergiogarciadev/ctmon/blob/e4a4f67f4b405821a2ab47ab1878d6ae0eebb72c/logclient/log.go#L92)   |
+|[Scrape CT Log](https://github.com/mpalmer/scrape-ct-log)                                              |files (json, cbor)           |[yes](https://github.com/mpalmer/scrape-ct-log/blob/02314930ac59c23f6b0782fe156239aeff86b667/src/runner/mod.rs#L72)                      |[yes](https://github.com/mpalmer/scrape-ct-log/blob/02314930ac59c23f6b0782fe156239aeff86b667/src/fetcher/mod.rs#L246)             |[randomized](https://github.com/mpalmer/scrape-ct-log/blob/02314930ac59c23f6b0782fe156239aeff86b667/src/fetcher/mod.rs#L183)|
+|[crt.sh](https://github.com/crtsh)                                                                     |SQL                          |[yes](https://github.com/crtsh/ct_monitor/blob/174e0d8d4954dacd80eaf45dedd90061d7e7a6f4/ct/logList.go#L24)                               |[yes](https://github.com/crtsh/ct_monitor/blob/174e0d8d4954dacd80eaf45dedd90061d7e7a6f4/ct/getEntries.go#L77)                     |[static](https://github.com/crtsh/ct_monitor/blob/174e0d8d4954dacd80eaf45dedd90061d7e7a6f4/ct/logList.go#L75)               |
+|[CertStream](https://github.com/CaliDog/certstream-server?tab=readme-ov-file)                          |files (json), last 25 entries|[yes](https://github.com/CaliDog/certstream-server/blob/41c054704316f9ade21a0cc89db19d51e10469e6/lib/certstream/ct_watcher.ex#L165)      |[no](https://github.com/CaliDog/certstream-server-python/blob/790718da384d3710e7842bd32b8367d2e142cc14/certstream/watcher.py#L143)|no                                                                                                                          |
+
+
+If you know of any other tool, or spot any error in this table, feel free to send a PR!
+
+# Online services 
+If you do not want to run a service yourself, various online service will fetch the data and make it available through
```

Comment:
"If you don't want to run your own log downloading and indexing system, there exist various online services which do this and offer the results via API or HTTP UIs, Let's Encrypt has a list [here]"?